src/python/scripts/concat_constrained_elems.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Concatenate the constrained elements from each GAB into one file for each species present in the
+alignment
+
+The coordinates of each constrained element are translated from the GAB coordinate level to the Genome
+coordinate level of each species and then they are concatenated into one file per species
+
+Typical usage example:
+
+    $ python concatenate_constrained_elements.py --json_desc gab_ce.txt --out_dir ./out/
+
+"""
+
+import argparse
+import json
+from typing import Dict
+
+from ensembl.compara.utils import alignment_to_seq_region
+
+
+def translate_ce_coordinates(json_conf: str) -> Dict:
+    """Project each constrained elements to the unaligned genomic level in each species contained in the
+    alignment
+
+    Params:
+        json_conf: file storing the list of json files to analyse
+
+    Returns:
+        a dictionary that has for key the assembly name and the value is the list of constrained elements at
+        the genomic coordinate level for this assembly a constraint element is a list of the type
+        [seq_name, start, end, score, pval, strand]
+    """
+    result = {}  # type: ignore
```

Comment:
This should work so don't have to ignore :crossed_fingers: 
```suggestion
    result = dict()
```

Reply:
I will give it a try but I do not see why it should change something as = {} and = dict() are equivalent.

Reply:
in fact, it does not work either we have to put the type: ignore tag